### PR TITLE
feat: lo-fi login flow

### PR DESCRIPTION
**Pull request by carolynzhuang**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Added basic login that leads you to the member login page (and guest login, which hasn't been created). 

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."
截屏 2024-10-17 18 37 32-1
截屏 2024-10-17 18 37 29

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
press the button that leads you from login to member login to ensure it works

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @christophertorres1

### CODE REVIEW

**Comment by katyhonguyen** [comment]
looks good to me, the code is clean!

File: src/navigation/types.tsx
Comment (christophertorres1) [inline src/navigation/types.tsx:1]: Rename the folder 'navigation' to 'types', and the file 'types.tsx' to 'types.ts'

File: src/components/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <View style={styles.buttonsContainer}>
+        <TouchableOpacity style={styles.button}>
+          <Text style={styles.buttonText}>Guest</Text>
+        </TouchableOpacity>
+
+        <TouchableOpacity
+          style={styles.button}
+          onPress={() => navigation.navigate('MemberLogin')}
+        >
+          <Text style={styles.buttonText}>Staff or Member</Text>
```
Comment (christophertorres1) [inline src/components/Login.tsx:21]: Change "Staff or Member" to "Admin"

File: src/screens/MemberLoginScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Text, TouchableOpacity, View } from 'react-native';
+import Svg, { Circle } from 'react-native-svg';
+import { styles } from '@/screens/styles';
+
+export default function MemberLoginScreen() {
+  return (
+    <View style={styles.memberLoginContainer}>
+      <Text style={styles.loginHeadingText}>Staff and Member Login</Text>
```
Comment (christophertorres1) [inline src/screens/MemberLoginScreen.tsx:8]: Change to "Admin Login"

**Review by christophertorres1 — CHANGES_REQUESTED** [review]
A few naming edits to ensure consistency throughout the project. Great job overall Carolyn!!

**Review by christophertorres1 — APPROVED** [review]
